src/radroutes.py

```diff
@@ -1,14 +1,92 @@
 #!/usr/bin/env python 
 
 import sys
+import os
+from schema import Schema, SchemaError, And
+import yaml
+import fit2gpx
+import folium
+from folium import plugins
+import pandas as pd
 
 class RadRouter():
-    def process(self, path_to_config):
-        """Consumes a path to a config file and generates a map."""
-        print(f"Path to config file: {path_to_config}")
-        return path_to_config
+    def process_activities(self, path_to_config):
+        # Consumes a path to a config file and generates a map.
+        activities = self.validate_config(path_to_config)
+
+        segments = activities['activities']['segments']
+        directory = activities['activities']['directory']
+
+        conv = fit2gpx.Converter()
+        all_coordinates_list = []
+        map = folium.Map()
+
+        # convert each file to a dataframe and create a list of dataframes
+        for segment in segments:
+            segment_path = directory + segment['file_name']
+            df_laps, df_coordinates = conv.fit_to_dataframes(segment_path)
```

Comment:
If `df_laps` is not being used, suggest to assign the return value to `_, df_coordinates` as an indication that this value is intentionally not being used.